### ISSUE TITLE
uknown tag load

### ISSUE THREAD

**Issue by mgutz** [cookiecutter/cookiecutter-django#6]
``` python
Cloning into 'cookiecutter-dj-project'...
remote: Counting objects: 441, done.
remote: Compressing objects: 100% (227/227), done.
remote: Total 441 (delta 196), reused 437 (delta 196)
Receiving objects: 100% (441/441), 118.47 KiB | 0 bytes/s, done.
Resolving deltas: 100% (196/196), done.
Checking connectivity... done
project_name (default is "project_name")?
repo_name (default is "repo_name")?
author_name (default is "Your Name")?
email (default is "Your email")?
description (default is "A short description of the project.")?
year (default is "Current year")?
domain_name (default is "Domain name")?
Traceback (most recent call last):
  File "/Users/mgutz/.virtualenvs/foo/bin/cookiecutter", line 9, in <module>
    load_entry_point('cookiecutter==0.6.4', 'console_scripts', 'cookiecutter')()
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/cookiecutter/main.py", line 90, in main
    cookiecutter(args.input_dir)
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/cookiecutter/main.py", line 61, in cookiecutter
    context=context
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/cookiecutter/generate.py", line 135, in generate_files
    generate_file(infile, context, env)
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/cookiecutter/generate.py", line 85, in generate_file
    tmpl = env.get_template(infile_fwd_slashes)
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/jinja2/environment.py", line 791, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/jinja2/environment.py", line 765, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/jinja2/environment.py", line 554, in compile
    self.handle_exception(exc_info, source_hint=source)
  File "/Users/mgutz/.virtualenvs/foo/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "./cookiecutter-dj-project/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/avatar/add.html", line 2, in template
    {% load i18n avatar_tags %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'load'.
```

**Comment by nitin42** [comment]
Cloning into 'cookiecutter-django'...
remote: Counting objects: 5752, done.
remote: Total 5752 (delta 0), reused 0 (delta 0), pack-reused 5751
Receiving objects: 100% (5752/5752), 2.46 MiB | 134.00 KiB/s, done.
Resolving deltas: 100% (3638/3638), done.
Checking connectivity... done.
project_name [project_name]: icecreamratings
repo_name [icecreamratings]: icecreamratings_project
author_name [Your Name]: nitin
email [Your email]: tulswani19@gmail.com
description [A short description of the project.]: site for rating the icecream flavours
domain_name [example.com]: icecream.nitin.com
version [0.1.0]: 0.1.0
timezone [UTC]: America/Los_Angeles
Traceback (most recent call last):
  File "/usr/bin/cookiecutter", line 9, in <module>
    load_entry_point('cookiecutter==1.3.0', 'console_scripts', 'cookiecutter')()
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/cookiecutter/cli.py", line 100, in main
    config_file=user_config
  File "/usr/lib/python2.7/dist-packages/cookiecutter/main.py", line 131, in cookiecutter
    context['cookiecutter'] = prompt_for_config(context, no_input)
  File "/usr/lib/python2.7/dist-packages/cookiecutter/prompt.py", line 126, in prompt_for_config
    val = render_variable(env, raw, cookiecutter_dict)
  File "/usr/lib/python2.7/dist-packages/cookiecutter/prompt.py", line 86, in render_variable
    template = env.from_string(raw)
  File "/usr/lib/python2.7/dist-packages/jinja2/environment.py", line 862, in from_string
    return cls.from_code(self, self.compile(source), globals, None)
  File "/usr/lib/python2.7/dist-packages/jinja2/environment.py", line 565, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/usr/lib/python2.7/dist-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "<unknown>", line 1, in template
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'now'.


still getting error!

**Comment by pydanny** [comment]
Update your version of Cookiecutter.

**Comment by nitin42** [comment]
Fixed. Thanks a lot!

**Comment by nitin42** [comment]
(ENV) nitin@nitin-Satellite-Pro-B40-A:~/ENV/site$ python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/nitin/ENV/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/home/nitin/ENV/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 303, in execute
    settings.INSTALLED_APPS
  File "/home/nitin/ENV/local/lib/python2.7/site-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/home/nitin/ENV/local/lib/python2.7/site-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/nitin/ENV/local/lib/python2.7/site-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/nitin/ENV/site/config/settings/local.py", line 11, in <module>
    from .common import *  # noqa
  File "/home/nitin/ENV/site/config/settings/common.py", line 13, in <module>
    import environ
ImportError: No module named environ

I tried all the tweaks but I am not able to resolve this. HELP!!??

**Comment by luzfcb** [comment]
this error happened because you probably do not install all dependencies in virtualenv. more specifically, are missing install django-environ package.

to be more practical and make sure everything is installed, install all dependencies in virtualenv. First, load your virtualenv, and inside the project root path, run:

```bash
pip install -r requirements/local.txt
pip install -r requirements/test.txt
pip install -r requirements/production.txt
```

ps .: Make sure you have previously installed all non-Python dependencies listed in: https://github.com/pydanny/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/utility/requirements.apt

**Comment by nitin42** [comment]
(ENV) nitin@nitin-Satellite-Pro-B40-A:~/ENV/mysite$ pip install -r requirements/production.txt
Requirement already satisfied (use --upgrade to upgrade): wheel==0.29.0 in /home/nitin/ENV/lib/python2.7/site-packages (from -r requirements/base.txt (line 5))
Collecting django==1.9.5 (from -r requirements/base.txt (line 8))
  Using cached Django-1.9.5-py2.py3-none-any.whl
Collecting django-environ==0.4.0 (from -r requirements/base.txt (line 11))
Collecting django-secure==1.0.1 (from -r requirements/base.txt (line 12))
  Using cached django-secure-1.0.1.tar.gz
Collecting django-braces==1.8.1 (from -r requirements/base.txt (line 17))
  Using cached django_braces-1.8.1-py2.py3-none-any.whl
Collecting django-crispy-forms==1.6.0 (from -r requirements/base.txt (line 18))
  Using cached django_crispy_forms-1.6.0-py2-none-any.whl
Collecting django-floppyforms==1.6.2 (from -r requirements/base.txt (line 19))
  Using cached django-floppyforms-1.6.2.tar.gz
Collecting django-model-utils==2.5 (from -r requirements/base.txt (line 22))
  Using cached django-model-utils-2.5.tar.gz
Collecting Pillow==3.2.0 (from -r requirements/base.txt (line 25))
  Using cached Pillow-3.2.0.zip
Collecting django-allauth==0.25.2 (from -r requirements/base.txt (line 29))
  Using cached django-allauth-0.25.2.tar.gz
Collecting psycopg2==2.6.1 (from -r requirements/base.txt (line 33))
  Using cached psycopg2-2.6.1.tar.gz
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found
    
    Error: pg_config executable not found.
    
    Please add the directory containing pg_config to the PATH
    or specify the full executable path with the option:
    
        python setup.py build_ext --pg-config /path/to/pg_config build ...
    
    or with the pg_config option in 'setup.cfg'.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-6rifqS/psycopg2/

Now I am stucked with this! I created a new virtual environment , installed cookiecutter and django 1.8 in it. After this I executed this command 'cookiecutter https://github.com/pydanny/cookiecutter-django' to create my project template. After creating the template I executed the above three commands but after execution of every command, this error occurs.

What things could be possibly I am doing wrong?
